apps/web/src/state/settings/index.ts

```diff
@@ -0,0 +1,20 @@
+import { modules } from '@walless/ioc';
+import { type SettingDocument } from '@walless/store';
+import { appState } from 'state/app';
+
+const id = 'settings';
+
+export const settingsActions = {
+	setConfigs: (settings: SettingDocument) => {
+		if (settings._id) {
+			appState.settingConfig = settings.config;
+		}
+	},
+	setPrivacy: async ({ hideBalance }: { hideBalance: boolean }) => {
+		await modules.storage.upsert<SettingDocument>(id, async (doc) => {
+			doc.config.hideBalance = hideBalance;
+
+			return doc;
+		});
+	},
+};
```

Comment:
we don't want new setting state at all, even if you only define actions for it - define that actions inside appState.